readthedocs/doc_builder/environments.py

```diff
@@ -150,13 +150,7 @@ def run(self):
         if self.combine_output:
             stderr = subprocess.STDOUT
 
-        environment = {}
-        environment.update(self.environment)
-        environment['READTHEDOCS'] = 'True'
-        if self.build_env is not None:
-            environment['READTHEDOCS_VERSION'] = self.build_env.version.slug
-            environment['READTHEDOCS_PROJECT'] = self.build_env.project.slug
-            environment['READTHEDOCS_LANGUAGE'] = self.build_env.project.language
```

Comment:
Is there a reason we're not setting these?

Reply:
We already set them in https://github.com/readthedocs/readthedocs.org/blob/7212d6ff738b24a10fb0f4227d3fbdf69e5cab42/readthedocs/projects/tasks.py#L537